### PR TITLE
Upgrade OpenSSL to v1.1.1g (Linux)

### PR DESCRIPTION
**Pull request by kneth**

### CODE REVIEW

**Review by fronck — APPROVED** [review]
LGTM

**Review by blagoev — COMMENTED** [review]
if we are not using these urls
https://github.com/realm/realm-js/pull/3528/files#diff-0d2cbc1f11dc451c1f97e7233ccb6c48ee192e9c73ee79434ae71ddb0576391bR183


 then this is actually a flag so we better change it to 
`required.OPENSSL = true`

the usage is here

https://github.com/realm/realm-js/pull/3528/files#diff-0d2cbc1f11dc451c1f97e7233ccb6c48ee192e9c73ee79434ae71ddb0576391bR150

**Review by blagoev — APPROVED** [review]

File: scripts/download-realm.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ function getCoreRequirements(dependencies, options, required = {}) {
         case 'linux':
             const arch_string = options.arch == 'arm' ? '-armhf': '';
             required.CORE_ARCHIVE = `realm-core-${flavor}-v${dependencies.REALM_CORE_VERSION}-Linux${arch_string}-devel.tar.gz`;
-            required.OPENSSL = "https://static.realm.io/downloads/openssl/1.1.1b/Linux/x86_64/openssl.tgz";
+            required.OPENSSL = true;
```
Comment (RedBeard0531) [inline scripts/download-realm.js:183]: ditto line 212

Comment (blagoev) [reply scripts/download-realm.js:183]: https://github.com/realm/realm-js/pull/3533/files

**Pull request merged**